test/tests/fulltextTest.js

```diff
@@ -133,7 +134,7 @@ describe("Zotero.Fulltext", function () {
 			toSync.push({
 				item: pdfAttachment,
 				content: "Zotero [zoh-TAIR-oh] is a free, easy-to-use tool to help you collect, "
-					+ "organize, cite, and share your research sources.\n\n",
```

Comment:
The test was failing because the pdftotext inserts `\n` after `share` when recognizing test.pdf.